package.json

```diff
@@ -491,12 +518,12 @@
         "test": "npm run test-compile && node ./out/src/tests/runTests.js"
     },
     "extensionDependencies": [
+        "vscode.git",
         "ms-kubernetes-tools.vscode-kubernetes-tools"
     ],
     "devDependencies": {
         "@types/chai": "^4.3.16",
-        "@types/download": "^8.0.5",
-        "@types/glob": "^8.1.0",
+        "@types/decompress": "^4.2.7",
```

Comment:
Any idea why download and glob are removed, initially thought decompress takes cares of those but seems like they aren't.